angular.json

```diff
@@ -39,5 +39,8 @@
       }
     }
   },
-  "defaultProject": "notification"
+  "defaultProject": "notification",
+  "cli": {
+    "analytics": "6b63c3ac-e480-4085-8624-e6fb2554641d"
```

Comment:
@sri-pusuluri please revert this change
CC: @rajeevsathish

Reply:
updated